src/sass/app_2/_common/tabs.scss

```diff
@@ -0,0 +1,56 @@
+/* stylelint-disable color-no-hex */
+/* stylelint-disable number-max-precision */
+.tab-container {
+    overflow-x: auto;
+    overflow-y: hidden;
+
+    &.left {
+        text-align: left;
+    }
+    &.center {
+        text-align: center;
+    }
+    &.right {
+        text-align: right;
+    }
+    .tab-wrapper {
+        position: relative;
+        display: inline-block;
+        margin-bottom: 3px;
+        border-bottom: 1px solid #e9e9ed;
+        width: 100%;
+    }
+    .tab {
+        display: inline-block;
+        padding: 16px;
+        background: #fff;
+        color: #333333;
+        cursor: pointer;
+        font-weight: 300;
+
+        i {
+            padding-right: 8px;
+        }
+        span {
+            font-size: 1rem;
+        }
+        &:hover, &--active {
+            color: #2a3052;
+            font-weight: 400;
+        }
+    }
+    .tab_underline {
```

Comment:
Probably should rename this to `tab-underline`.

Reply:
yup.